api/Event.json

```diff
@@ -0,0 +1,1356 @@
+{
+  "api": {
+    "Event": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Event",
+        "support": {
+          "chrome": {
+            "version_added": "1"
+          },
+          "chrome_android": {
+            "version_added": "1"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "1"
```

Comment:
I'm seeing `(Yes)`  in the existing compat table, which I think should be tagged as`true`.